.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Description
+
+## Known Issues:
+
+### Checklist for this pull request
+
+* [ ] PR is scoped to one task
+* [ ] Tests included
+* One of:
+* [ ] The PR is not making any UI change
```

Comment:
I think you need to indent those
https://github.com/applicaster/zapp/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Can you give me access to that repo?

Reply:
you are now part of applicaster team, meaning you have read access on everything (plus write access on RN repos as member of team React-Native)